cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsUpdateService.java

```diff
@@ -48,9 +50,20 @@
         return cloudResourceStatuses;
     }
 
+    public void updateUserData(AuthenticatedContext authenticatedContext, CloudStack stack, List<CloudResource> resources, String userData) {
+        String cfTemplate = stack.getTemplate();
+        if (cfTemplate.contains("AWS::EC2::LaunchTemplate")) {
```

Comment:
could you move this string to a constant somewhere, as it's already present in another class?